templates/resources/common/modify-wrapper-linux-conf.gawk

```diff
@@ -37,3 +37,13 @@ END {
   printf("wrapper.java.additional.%d=-XX:+UseCGroupMemoryLimitForHeap\n", additionals + 2);
   printf("wrapper.java.additional.%d=-XX:MaxRAMFraction=2\n", additionals + 3);
 }
+
+# Enable JMX exporter for Prometheus
+END {
+  printf("wrapper.java.additional.%d=-Dcom.sun.management.jmxremote\n", additionals + 4);
+  printf("wrapper.java.additional.%d=-Dcom.sun.management.jmxremote.authenticate=false\n", additionals + 5);
+  printf("wrapper.java.additional.%d=-Dcom.sun.management.jmxremote.ssl=false\n", additionals + 6);
+  printf("wrapper.java.additional.%d=-Dcom.sun.management.jmxremote.port=9011\n", additionals + 7);
+  printf("wrapper.java.additional.%d=-Dcom.sun.management.jmxremote.rmi.port=9011\n", additionals + 8);
+  printf("wrapper.java.additional.%d=-javaagent:lib/jmx_prometheus_javaagent.jar=9100:conf/jmx-exporter.yaml\n", additionals + 9);
```

Comment:
What's the `9100` here?

Reply:
@hierynomus JMX-to-Prometheus exporter is exposed on this port. With this config It listens on 0.0.0.0:9100.

Reply:
Got it!